src/index.html

```diff
@@ -11,8 +11,59 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      data-qa="card"
+      class="card"
+    >
+      <img
+        src=".//images/imac.jpeg"
+        alt="iMac"
+        class="card__image"
+      />
+
+      <h1 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <h2 class="card__product-code">Product code: 195434</h2>
```

Comment:
Consider will be better if you use tag p instead of h2 everywhere
```suggestion
      <p class="card__product-code">Product code: 195434</p>
```